typescript/deploy/src/config.ts

```diff
@@ -12,7 +12,6 @@ export interface CheckerViolation {
 
 export type TransactionConfig = {
   overrides?: ethers.Overrides;
```

Comment:
I think we now need to update the TransactionConfigs in [this file](https://github.com/abacus-network/abacus-monorepo/blob/main/typescript/infra/config/networks/mainnets.ts#L10). According to this comment https://github.com/ethers-io/ethers.js/issues/1610#issuecomment-915460497, if the overrides provide explicit gas fee parameters, ethers skips the automatic checking. I think at the least we'll want to probably get rid of `gasPrice` for the networks that we know support EIP 1559?